plugins/home/README.md

```diff
@@ -325,6 +325,34 @@ export default app.createRoot(
 );
 ```
 
+You can filter the items that are shown in the component.
+this can be done by using the config file.
+Filtering is done by using 3 parameters:
+
+- `field` - define which field to filter. can be one of the following
+  - id: string
+  - name: string
+  - pathname: string
+  - hits: number
+  - timestamp: number
+  - entityRef: string
```

Comment:
Vale isn't happy with `entityRef`, if you put backticks (`) around it that is one way to fix it